amf-shapes/shared/src/main/scala/amf/shapes/internal/spec/raml/parser/Raml10TypeParser.scala

```diff
@@ -1605,7 +1605,13 @@ sealed abstract class RamlTypeParser(
 
       map.key("discriminator", (NodeShapeModel.Discriminator in shape).allowingAnnotations)
       map.key("discriminatorValue", (NodeShapeModel.DiscriminatorValue in shape).allowingAnnotations)
-
+      map.key(
+        "discriminatorValue",
```

Comment:
Look ups are constant time (map.entries is an IndexedSeq). Do you believe it's worth adding the extra lines of code rather than reusing the helper above?